dev/TEAL_opcodes.md

```diff
@@ -472,6 +472,8 @@ FirstValidTime causes the program to fail. The field is reserved for future use.
 | 7 | LatestTimestamp | uint64 | Last confirmed block UNIX timestamp. Fails if negative. LogicSigVersion >= 2. |
```

Comment:
Sometimes `panic` is used, sometimes `fail` is used.
We may want to make the use uniform, or have some text saying `panic` = `fail`

Reply:
I changed all "panic" wording to "fail" which seems clearer.